base/dcp_client.go

```diff
@@ -28,6 +28,10 @@ const openStreamTimeout = 30 * time.Second
 const openRetryCount = uint32(10)
 const defaultNumWorkers = 8
 
+// DCP buffer size values based off if we are running in serverless or not
+const DefaultDCPBufferServerless = 1 * 1024 * 1024
+const DefaultDCPBuffer = 20 * 1024 * 1024
```

Comment:
Similar to the above - we don't want to explicitly set the value unless we're changing it.